### PR TITLE
more tests for federation sharing

### PR DESCRIPTION
**Pull request by individual-it**
## Description
some more tests to check if federation sharing works correctly

## Motivation and Context
manly to make sure #30106 does not break anything

## How Has This Been Tested?
run tests locally, now let :robot: work

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/bootstrap/WebUIFilesContext.php [hunk 1]
```diff
@@ -1042,7 +1101,7 @@ public function theFollowingFileFolderShouldBeListedInTheFollowingFolderOnTheWeb
 	}
 
 	/**
-	 * @Then /^the following (?:file|folder) should (not|)\s?be listed\s?(?:in the |)(trashbin|favorites page|)\s?(?:folder ((?:'[^']*')|(?:"[^"]*")))? on the webUI$/
+	 * @Then /^the following (?:file|folder) should (not|)\s?be listed\s?(?:in the |)(trashbin|favorites page|shared-with-you page|)\s?(?:folder ((?:'[^']*')|(?:"[^"]*")))? on the webUI$/
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIFilesContext.php]: We could add "files page" in this step also, to be consistent with the step at line 943. (It was not here yet, but was above - some inconsistenct=y from the past)

**Review by phil-davis — DISMISSED** [review]
Looks good. It will be nice keeping track of the current server that the webUI is logged in to, as a different thing to the current server that the API steps are using.

**Comment by individual-it** [comment]
fix conflicts and added "files page" to `theFollowingFileFolderShouldBeListedOnTheWebUI()` please review again

**Review by phil-davis — APPROVED** [review]
:+1:

**Comment by phil-davis** [comment]
Rebase and force push done to waken CI.

**Comment by phil-davis** [comment]
"random" webdriver timeout in webUIRenameFolders suite. drone has been restarted.

**Comment by phil-davis** [comment]
Backport ``stable10`` #31143

**Pull request merged**